plugins/include/amxconst.inc

```diff
@@ -44,6 +49,11 @@
  */
 public stock const MaxClients; 
 
+/**
+ * Current map name
```

Comment:
Please add to the description is it map file path (`maps/de_dust2.bsp`) or just a map name (`de_dust2`).

Reply:
@WPMGPRoSToTeMa if this were the path to the map, I would point out that this is the path.